composer.json

```diff
@@ -12,15 +12,17 @@
     ],
     "require": {
         "php": ">=7.1",
-        "keboola/db-extractor-common": "~9.1",
+        "keboola/db-extractor-common": "dev-ondra-severance-repository@dev",
+        "keboola/db-extractor-logger": "dev-ondra-init-repo@dev",
+        "keboola/db-extractor-ssh-tunnel": "dev-ondra-init-repo@dev",
```

Comment:
Snowflake navíc nemá SSH tunel, ne?

Reply:
nemáá, ale je to dependency v db-ex-common a kvůli min-stability jsem to musel přidat i sem

až se to bude requirovat do produkce tak to tu potřeba nebude stejně jako `db-ex-logger`

Reply:
Jasně!